### ISSUE TITLE
REGRESSION - default user groupd on new category and/or content

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#7311]
I have not been able to replicate this myself but there are multiple reports on the forum.

Basically when creating a new article or category the access is being set to Guest and not public.

See http://forum.joomla.org/viewtopic.php?f=706&t=890720&p=3314030#p3314030

**Comment by nonumber** [comment]
PS: this concerns the default access level, not user group.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7311">issues.joomla.org/joomla-cms/7311</a>.</sub>

**Comment by zero-24** [comment]
hmm maybe a `pre select`? So that the access level is set per search tools? That would explain why you can't reproduce :smile: 

PR was: https://github.com/joomla/joomla-cms/pull/6966

**Comment by infograf768** [comment]
I confirm the issue for New article and New category. Nothing selected in Search Tools.

**Comment by nonumber** [comment]
Commenting out that line seems to fix the isse:
```$data->set('access', $app->input->getInt('access', (isset($filters['access']) ? $filters['access'] : null)));```
in /administrator/components/com_content/models/article.php

**Comment by Bakual** [comment]
To reproduce change the default access level to something else.

**Comment by Bakual** [comment]
The problem is that that PR added the property and set it to `null`. It should probably instead set it to `JFactory::getConfig()->get('access')` similar to what JTable does: https://github.com/joomla/joomla-cms/blob/staging/libraries/joomla/table/table.php#L182-L186.
Or just leave it alone and let JTable handle it.

**Comment by nonumber** [comment]
I think it would be best to remove that line all together.
Otherwise this would fix it:
```
$data->set('access', $app->input->getInt('access', ((isset($filters['access']) && $filters['access'] != '') ? $filters['access'] : JFactory::getConfig()->get('access'))));
```

**Comment by Bakual** [comment]
See https://github.com/joomla/joomla-cms/pull/7313

Closing as we have a PR.

**Comment by prankyboy** [comment]
Anche creando nuovo modulo html l'accesso risulta solo per guest<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7311">issues.joomla.org/joomla-cms/7311</a>.</sub>

**Comment by nonumber** [comment]
Also fixed with PR #7313
